project4/project/ebs/templates/ebs/create_blog.html

```diff
@@ -0,0 +1,257 @@
+						<img class="center-block img-responsive" src="{% static 'ebs/img/logo-footer.png' %}"/>
+					</div>
+				</div>
+				<div class="row">
+					<div class="col-md-offset-2 col-md-8 col-xs-offset-2 col-xs-8 footer-padding">
+						<span>Newspaper is your news, entertain, music fashion website. We provide you with the latest breaking news and videos straight from the entertainment industry.</span>
+					</div>
+				</div>
+
+				<div class="row">
+					<div class="col-md-12 text-center footer-padding-2">
+						<span>Contact us:<a style="color:#009688" href="#"> contact@yourside.com</a></span>
+					</div>
+				</div>
+
+				<div class="row">
+					<div class="col-md-12 text-center footer-padding-2">
+						<a href="#"><i class="fa fa-facebook-square fa-2x footer-icon"></i></a>
+						<a href="#"><i class="fa fa-twitter-square fa-2x footer-icon"></i></a>
+						<a href="#"><i class="fa fa-linkedin-square fa-2x footer-icon"></i></a>
+					</div>
+				</div>
+
+			</div>
+		</footer>
+
+
+
+	<footer id="footer-2">
+		<div class="container">
+			<div class="row">
+				<div class="col-md-6">
+					<ul class="footer2-padding">
+						<li><a href="#">Contact us</a></li>
+						<li><a href="#"> About us</a></li>
+						<li><a href="#"> Blog</a></li>
+					</ul>
+				</div>
+
+				<div class="col-md-6 copyright">
+					<span > &COPY; Copyright 2017 - Newspaper WordPress Theme by TagDiv</span>
+				</div>
+
+			</div>
+
+		</div>
+	</footer>
+
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"> </script>
+<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+<script src="{% static 'ebs/bootstrap-3.3.7-dist/js/bootstrap.min.js' %}"></script>
+<script src='http://cdnjs.cloudflare.com/ajax/libs/bootstrap-validator/0.4.5/js/bootstrapvalidator.min.js'></script>
+<script src="http://code.jquery.com/jquery-1.9.1.js"></script>
+<script src="http://code.jquery.com/ui/1.11.0/jquery-ui.js"></script>
+<script type="text/javascript" src="{% static 'ebs/js/notify.min.js' %}"></script>
+
+<!--<script src="https://cdn.jsdelivr.net/jquery.validation/1.15.0/additional-methods.min.js"></script>-->
+<!--<script type="text/javascript" src="{% static 'ebs/js/registration.js' %}"></script>
+<script type="text/javascript" src="{% static 'ebs/js/registration1.js' %}"></script>
+<script type="text/javascript" src="{% static 'ebs/js/login.js' %}"></script>
+<script type="text/javascript" src="{% static 'ebs/js/forgotpassword.js' %}"></script>
+<script type="text/javascript" src="{% static 'ebs/js/resetpassword.js' %}"></script>-->
+<!--<script>
+$(document).ready(function() {
+    $('.datepicker').datepicker();
+});-->
+
+  <!--{{ blogform.media.js }}-->
+<!--<script type="text/javascript">
+            $(function () {
+                $('#datetimepicker1').datetimepicker({
+                    
+                });
+            });
+        </script>
+</script>
+-->
+</body>
+</html>
```

Comment:
Header, navigation and footer sections should be in a common file, that should not repeat in very templates. Need to create new task in next sprint.